rundeckapp/grails-spa/packages/ui/src/pages/project-nodes-config/ProjectPluginConfig.vue

```diff
@@ -449,6 +493,44 @@ export default Vue.extend({
     },
     configUpdated() {
       this.pluginConfigsModified();
+    },
+    hasKeyStorageAccess(provider: any){
+      this.pluginStorageAccess.push(provider)
+
+    },
+    cleanStorageAccess(plugin: any){
+      const pluginStorageAccess = [] as any[];
+      this.pluginStorageAccess.forEach((pluginAccess:any)=>{
+        if(pluginAccess !== plugin.entry.type){
+          pluginStorageAccess.push(pluginAccess);
+        }
+      });
+
+      this.pluginStorageAccess = pluginStorageAccess;
+    },
+    openCreateAcl(){
+      this.modalAclOpen = true;
+    },
+    async createAcl(){
+      if(this.aclDescription !== ''){
+        try{
+          await createProjectAcl(this.aclDescription );
```

Comment:
I think this is too opaque to the end user, they should have a chance to see the ACL yaml and modify it.  I would rather the "Create ACL" button takes them to the Access Control page with a prefilled ACL and let's them save it manually